test/correctness/simd_op_check.h

```diff
@@ -0,0 +1,301 @@
+#ifndef SIMD_OP_CHECK_H
+#define SIMD_OP_CHECK_H
+
+#include "Halide.h"
+#include "test/common/halide_test_dirs.h"
+#include "sys/time.h"
+#include <fstream>
+
+using namespace Halide;
+using namespace Halide::ConciseCasts;
+
+//namespace {
+
+// This tests that we can correctly generate all the simd ops
+using std::vector;
```

Comment:
You should ~never put `using` statements in the toplevel namespace in a .h file like this

Reply:
Fixed.